build.sbt

```diff
@@ -74,7 +74,7 @@ lazy val root = (project in file("."))
         "com.dwolla" %% "scala-cloudformation-custom-resource" % "2.1.0",
         "com.dwolla" %% "fs2-aws" % "1.1.0",
         "io.circe" %% "circe-fs2" % "0.9.0",
-        "com.dwolla" %% "cloudflare-api-client" % "3.0.1",
+        "com.dwolla" %% "cloudflare-api-client" % "4.0.0-M2-SNAPSHOT",
```

Comment:
This is wrong, will need to publish the latest scala-cloudflare then I can update.

Reply:
This is causing the Travis CI build to fail. I wanted to get this PR out even though its failing to get some eyes on it.

Reply:
`scala-cloudflare` has been merged in and this should now be the appropriate version.